### PR TITLE
Supress logging within `writeproto` to prevent infinite recursion.

### PR DESCRIPTION
**Pull request by JJMinton**
This PR addresses https://github.com/PhilipVinc/TensorBoardLogger.jl/issues/96, following the suggestion of @oxinabox.

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -17,3 +17,13 @@ function serialize_proto(data::Any)
     write(pb, data)
     pb.data
 end
+
+"""
+    wrapper for writeproto that supresses logging to prevent infinite
+    recursion.
+"""
+function logless_writeproto(pb::IO, obj)
```
Comment (PhilipVinc) [inline src/utils.jl]: Can you call this simply `_writeproto`? . Leave the docstring the same.

**Comment by PhilipVinc** [comment]
CI is not running even if I approved it.
Can you try pushing another commit? (you can bump the patch version, which I'd need to do anyway)

**Comment by JJMinton** [comment]
Thanks for the review and merging @PhilipVinc. Glad to get it sorted.

**Comment by PhilipVinc** [comment]
Thanks to you, actually.
Sorry I'm super busy those days so I can barely keep up with OSS.

Anyhow, I wanted to tag a new release, but It seems to me that the test you added is a bit flaky (not more than GitHub Actions have been those last few days, though).
It's [currently failing](https://github.com/PhilipVinc/TensorBoardLogger.jl/runs/2614157862#step:6:65) on Master.
Could you please give a look?

**Comment by PhilipVinc** [comment]
I don't understand at a glance how this can fail somewhere and pass somewhere else...

**Pull request merged**